docker-compose.yaml

```diff
@@ -229,11 +229,12 @@ services:
     ports:
       - "127.0.0.1:8147:8547"
       - "127.0.0.1:8148:8548"
+      - "127.0.0.1:9800:9800"
     volumes:
       - "poster-data:/home/user/.arbitrum/local/nitro"
       - "l1keystore:/home/user/l1keystore"
       - "config:/config"
-    command: --conf.file /config/poster_config.json
+    command: --conf.file /config/poster_config.json --node.feed.output.enable --node.feed.output.port 9800
```

Comment:
I don't like having feed output from batch_poster. sequencer is better because it exists in every architecture.

Reply:
Agreed, removed :)